packages/tests/tests/store/utils.ts

```diff
@@ -18,12 +18,20 @@ export const log = new Logger("test:store");
 export const TestContentTopic = "/test/1/waku-store/utf8";
 export const TestEncoder = createEncoder({ contentTopic: TestContentTopic });
 export const TestDecoder = createDecoder(TestContentTopic);
-export const customContentTopic = "/test/2/waku-store/utf8";
-export const customPubSubTopic = "/waku/2/custom-dapp/proto";
-export const customTestDecoder = createDecoder(
-  customContentTopic,
-  customPubSubTopic
-);
+export const customShardedPubsubTopic1 = "/waku/2/rs/3/1";
```

Comment:
Let's not have such generic names. If it is intended to be re-used let's name just `customShardedPubsubTopic`
Otherwise let's create them ad-hoc in a test once needed.
Or let's have an array of topics/decoders/etc to re-use later.